File: Codeforces Problems/BunLover.java
import java.util.*;

public class BunLover {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int t;
        t = sc.nextInt();

        for (int i = 0; i < t; i++) {
            int n, total = 0, sum = 0;

            n = sc.nextInt();

            total = 4 * n;
            total += n;
            sum = (n - 1) * (n - 2);
            total += sum;
            System.out.println(total + "\n");
        }

    }
}

File: Codeforces Problems/bun_lover.c
#include <stdio.h>
int main()
{
    int t;
    scanf("%d", &t);

    for (int i = 0; i < t; i++)
    {
        long long int n, total = 0, sum = 0;
        scanf("%lld", &n);

        total = 4 * n;
        total += n;
        sum = (n - 1) * (n - 2);
        total += sum;

        printf("%lld\n", total);
    }
}